src/Makefile.qt.include

```diff
@@ -597,18 +597,18 @@ RES_IMAGES = \
   qt/res/images/ic_solo_inactive.svg \
   qt/res/images/splash3.png
 
-RES_FONTS = \
+QT_RES_FONTS = \
   qt/res/fonts/Inconsolata-Regular.ttf \
   qt/res/fonts/Inter-Bold.otf \
   qt/res/fonts/Inter-Medium.otf \
   qt/res/fonts/Inter-Regular.otf
 
-RES_STYLESHEETS = \
+QT_RES_STYLESHEETS = \
   qt/res/stylesheets/light_stylesheet.qss \
   qt/res/stylesheets/dark_stylesheet.qss
 
 
-GRIDCOIN_RC = qt/res/gridcoinresearch.rc
+GRIDCOIN__QT_RC = qt/res/gridcoinresearch.rc
```

Comment:
> GRIDCOIN__QT_RC
Here seems to have been inserted a underscore too much?

Reply:
I agree. Not sure how it is working with that...

Reply:
> > GRIDCOIN__QT_RC
> 
> Here seems to have been inserted a underscore too much?

Good catch, thanks!